### PR TITLE
Add roles for volume-delete

### PR DESCRIPTION
**Pull request by devyanikota**
Signed-off-by: Devyani Kota <devyanikota@gmail.com>

### CODE REVIEW

**Comment by devyanikota** [comment]
@sac can you please review this. thanks :)

File: roles/glusterfs/volume-delete/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+volume-create
+=============
```
Comment (sac) [inline roles/glusterfs/volume-delete/README.md]: Isn't this volume-delete?

File: roles/glusterfs/volume-delete/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Creates a GlusterFS volume. Volume can be of any type - replicate, distributed-replicate, arbiter ...
+
```
Comment (sac) [inline roles/glusterfs/volume-delete/README.md]: Same as above. I think you haven't modified the documentation at all.

File: roles/glusterfs/volume-delete/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (sac) [inline roles/glusterfs/volume-delete/defaults/main.yml]: These are not the right role-variable(s)

File: roles/glusterfs/volume-delete/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (sac) [inline roles/glusterfs/volume-delete/tasks/main.yml]: We don't need all these. Only volume name and state as absent.

File: roles/glusterfs/volume-delete/README.md (hunk 2 above)
Comment (devyanikota) [reply roles/glusterfs/volume-delete/README.md]: Made changes to the documentation adding volume-delete role specifications.

**Comment by sac** [comment]
Moved to gluster.cluster

**Pull request closed without merging**